src/components/Filters/Filters-container.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import Grid from '@material-ui/core/Grid/Grid';
+
+import useFiltersStyles from './Filters-styles';
+
+const Filters = () => {
+  const classes = useFiltersStyles();
+  const blockchains: string[] = ['BTC', 'ETH', 'XRP', 'LTC', 'BCH', 'XTZ', 'ADA', 'EOS', 'XLM'];
+  const activeClasses = [classes.button, classes.active]
+
+  const filterHandler = (eventTarget: HTMLBodyElement) => {
+    const active: boolean = eventTarget.className === classes.button ? true : false;
+    active && (eventTarget.className = activeClasses.join(' '));
+    !active && (eventTarget.className = classes.button);
+  }
+
+  const renderButtons = (data: string[]) =>
+    data.map((button: string, index: number) => (
```

Comment:
Generalnie w mapie nie powinno się używać indexu jako klucza. AirBnB ma chyba na to nawet specjalną regułę lintową. Oczywiście jeśli nie innej właściwości, której można użyć, to pewnie musi ten index zostać.

Reply:
Podpiełam się pod nazwę buttona